### DISCUSSION TITLE
Broken URLs

### DISCUSSION THREAD

**Discussion by InquilineKea** [modelcontextprotocol/.github#82] [General]
### Pre-submission Checklist

- [X] I have verified that this discussion would not be more appropriate as an issue in a specific repository
- [X] I have searched existing discussions to avoid duplicates

### Discussion Topic

https://modelcontextprotocol.io/src/gdrive is broken
as is https://modelcontextprotocol.io/src/filesystem
[all from https://modelcontextprotocol.io/examples ]

**Comment by jspahrsummers** [comment]
Thanks, I think this is fixed now. If there are any other problems you notice in the docs, please do open issues on the `docs` repo. Cheers!
